gcloud/monitoring/client.py

```diff
@@ -0,0 +1,189 @@
+                 ``credentials`` for the current object.
+    """
+
+    _connection_class = Connection
+
+    def query(self,
+              metric_type=Query.DEFAULT_METRIC_TYPE,
+              end_time=None,
+              days=0, hours=0, minutes=0):
+        """Construct a query object for listing time series.
+
+        Example::
+
+            >>> query = client.query(minutes=5)
+            >>> print query.as_dataframe()  # Requires pandas.
+
+        :type metric_type: string
+        :param metric_type: The metric type name. The default value is
+            ``"compute.googleapis.com/instance/cpu/utilization"``, but
+            please note that this default value is provided only for
+            demonstration purposes and is subject to change. See the
+            `supported metrics`_.
+
+        :type end_time: :class:`datetime.datetime` or None
+        :param end_time: The end time (inclusive) of the time interval
+            for which results should be returned, as a datetime object.
+            The default is the start of the current minute.
+
+            The start time (exclusive) is determined by combining the
+            values of  ``days``, ``hours``, and ``minutes``, and
+            subtracting the resulting duration from the end time.
+
+            It is also allowed to omit the end time and duration here,
+            in which case
+            :meth:`~gcloud.monitoring.query.Query.select_interval`
+            must be called before the query is executed.
+
+        :type days: integer
+        :param days: The number of days in the time interval.
+
+        :type hours: integer
+        :param hours: The number of hours in the time interval.
+
+        :type minutes: integer
+        :param minutes: The number of minutes in the time interval.
+
+        :rtype: :class:`~gcloud.monitoring.query.Query`
+        :returns: The query object.
+
+        :raises: :exc:`ValueError` if ``end_time`` is specified but
+            ``days``, ``hours``, and ``minutes`` are all zero.
+            If you really want to specify a point in time, use
+            :meth:`~gcloud.monitoring.query.Query.select_interval`.
+
+        .. _supported metrics: https://cloud.google.com/monitoring/api/metrics
+        """
+        return Query(self, metric_type,
+                     end_time=end_time,
+                     days=days, hours=hours, minutes=minutes)
+
+    def fetch_metric_descriptor(self, metric_type):
+        """Look up a metric descriptor by type.
+
+        Example::
+
+            >>> METRIC = 'compute.googleapis.com/instance/cpu/utilization'
+            >>> print client.fetch_metric_descriptor(METRIC)
+
+        :type metric_type: string
+        :param metric_type: The metric type name.
+
+        :rtype: :class:`~gcloud.monitoring.metric.MetricDescriptor`
+        :returns: The metric descriptor instance.
+
+        :raises: :class:`gcloud.exceptions.NotFound`
```

Comment:
Ditto here on the exception, though we are guilty of something like this from time to time. If you elect to keep this section, I'd suggest at least adding "if no results are found for the query".